src/selection-sharer.js

```diff
@@ -232,6 +232,7 @@
     this.shareFacebook = function(e) {
       e.preventDefault();
       var text = self.htmlSelection.replace(/<p[^>]*>/ig,'\n').replace(/<\/p>|  /ig,'').trim();
+      text = text.replace(/<\/?[^>]+(>|$)/g, ""); // Remove HTML tags, like <h3>foo</h3>bar => foobar
```

Comment:
Feel free to send a PR with the `textSelection` change if it's urgent. I think it's safe to say that `textSelection` is the better solution to this.